flake.nix

```diff
@@ -0,0 +1,382 @@
+              };
+              doCheck = false;
+              vendorHash = "sha256-jVOb2uHjPley+K41pV+iMPNx67jtb75Rb/ENhw+ZMoM=";
+            };
+
+            go-swagger = pkgs.buildGoModule rec {
+              pname = "go-swagger";
+              version = "0.28.0";
+
+              src = pkgs.fetchFromGitHub {
+                owner = "go-swagger";
+                repo = "go-swagger";
+                rev = "v${version}";
+                sha256 = "sha256-Bw84HQxrI8cSBEM1cxXmWCPqKZa5oGsob2iuUsiAZ+A=";
+              };
+              doCheck = false;
+              vendorHash = "sha256-lhb3tvwhTPNo5+OhGgc3p3ddxFtL1gaIVTpZw0krBhM=";
+            };
+
+            controller-tools = pkgs.buildGoModule rec {
+              pname = "controller-tools";
+              version = "0.4.1";
+
+              src = pkgs.fetchFromGitHub {
+                owner = "kubernetes-sigs";
+                repo = "controller-tools";
+                rev = "v${version}";
+                sha256 = "sha256-NQlSP9hRLXr+iZo0OeyF1MQs3PourQZN0I0v4Wv5dkE=";
+              };
+              vendorHash = "sha256-89hzPiqP++tQpPkcSvzc1tHxHcj5PI71RxxxUCgm0BI=";
+              doCheck = false;
+            };
+
+            k8sio-tools = pkgs.buildGoModule rec {
+              pname = "k8sio-tools";
+              version = "0.21.5";
+
+              src = pkgs.fetchFromGitHub {
+                owner = "kubernetes";
+                repo = "code-generator";
+                rev = "v${version}";
+                sha256 = "sha256-x05eAO2oAq/jm1SBgwjKo6JRt/j4eMn7oA0cwswLxk8=";
+              };
+              vendorHash = "sha256-Re8Voj2nO8geLCrbDPqD5rLyiUqE7APcgOnAEJzlkOk=";
+              doCheck = false;
+            };
+
+            goreman = pkgs.buildGoModule rec {
+              pname = "goreman";
+              version = "0.3.11";
+              src = pkgs.fetchFromGitHub {
+                owner = "mattn";
+                repo = "goreman";
+                rev = "v${version}";
+                sha256 = "sha256-TbJfeU94wakI2028kDqU+7dRRmqXuqpPeL4XBaA/HPo=";
+              };
+              vendorHash = "sha256-87aHBRWm5Odv6LeshZty5N31sC+vdSwGlTYhk3BZkPo=";
+              doCheck = false;
+            };
+
+            stern = pkgs.buildGoModule rec {
+              pname = "stern";
+              version = "1.25.0";
+              src = pkgs.fetchFromGitHub {
+                owner = "stern";
+                repo = "stern";
+                rev = "v${version}";
+                sha256 = "sha256-E4Hs9FH+6iQ7kv6CmYUHw9HchtJghMq9tnERO2rpL1g=";
+              };
+              vendorHash = "sha256-+B3cAuV+HllmB1NaPeZitNpX9udWuCKfDFv+mOVHw2Y=";
+              doCheck = false;
+            };
+
+            staticfiles = pkgs.buildGoModule {
+              pname = "staticfiles";
+              version = "0.0.1"; # no official version
+              src = pkgs.fetchFromGitHub {
+                owner = "isubasinghe";
```

Comment:
Remove reference to personal repos

Reply:
I definitely agree that this is generally a bad idea, unfortunately the repo is dead. https://github.com/bouk/staticfiles. 

The current method to install this dependency is actually quite dangerous: https://github.com/argoproj/argo-workflows/blob/master/Makefile#L164. 

It uses an sha abbreviation instead of the full hash, on top of this it downloads off a random URL. 
We should get rid of this ASAP as it opens up the codebase to a code injection attack. 

I needed to fork his code because it was missing a [go.mod](https://github.com/isubasinghe/staticfiles/blob/master/go.mod). 

downloading from my repo is not great, but still safer than what the Makefile does. 

I will create an issue to get rid of this dep all together.